lib/libxslt.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="http://repo.roscidus.com/lib/libxslt" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>LibXslt</name>
+  <summary xml:lang="en">LibXslt: library and tools for applying XSLT to XML-documents</summary>
+  <description xml:lang="en">LibXslt is a library and tools for applying XSLT (Extensible Styleheets Transformations) to XML-documents. XML (Extensible Markup Language) is the universal format for data on the Web. XML is a metalanguage to design markup languages, i.e. text language where semantic and structure are added to the content using extra &quot;markup&quot; information enclosed between angle bracket. HTML is the most well-known markup language. </description>
+  <icon href="https://gitlab.com/pmiess/0install-feeds/raw/test/public/gnuwin32/libxml.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://gitlab.com/pmiess/0install-feeds/raw/test/public/gnuwin32/libxml.png" type="image/png"/>
+  <category>Development</category>
+  <homepage>http://gnuwin32.sourceforge.net/packages/libxslt.htm</homepage>
+  <needs-terminal/>
+  <implementation arch="Windows-*" id="sha1new=7bcb55f4519daf867981e4d03d042f4ac3d7f32a" license="GPL v2 (GNU General Public License), LGPL (GNU Lesser General Public License)" released="2001-12-23" version="1.0.9">
```

Comment:
For feeds providing 32-bit Windows executables without additional depenencies we usually use `arch="Windows-*"`. This works well because such executables work on all current Windows versions and can be used/called as depenencies by both 32bit and 64bit executables.

However, for feeds providing libraries (DLLs) it is better to specify the bitness (e.g. `i486`) explicitly, to prevent a 64bit EXE from trying (and failing) to load a 32bit DLL.

Reply:
Done